tests/test-dirs/search-by-type-comparison-to-polarity-search.t

```diff
@@ -132,17 +132,17 @@ map).
     "name": "Seq.map",
     "type": "('a -> 'b) -> 'a Stdlib__Seq.t -> 'b Stdlib__Seq.t"
   }
-  {
-    "name": "List.concat_map",
-    "type": "('a -> 'b list) -> 'a list -> 'b list"
-  }
   {
     "name": "List.filter_map",
     "type": "('a -> 'b option) -> 'a list -> 'b list"
   }
   {
-    "name": "ListLabels.concat_map",
-    "type": "f:('a -> 'b list) -> 'a list -> 'b list"
+    "name": "List.concat_map",
+    "type": "('a -> 'b list) -> 'a list -> 'b list"
+  }
+  {
+    "name": "ListLabels.filter_map",
+    "type": "f:('a -> 'b option) -> 'a list -> 'b list"
```

Comment:
Why did this test result change ?

Reply:
Because doc was removed (and it change the result)